opensrp-chw/src/nacp/assets/json.form/pnc_hygiene_observation.json

```diff
@@ -0,0 +1,169 @@
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "subscriberid",
+      "openmrs_entity_id": "163150AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "simserial": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "simserial",
+      "openmrs_entity_id": "163151AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "phonenumber": {
+      "openmrs_entity_parent": "",
+      "openmrs_entity": "concept",
+      "openmrs_data_type": "phonenumber",
+      "openmrs_entity_id": "163152AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+    },
+    "encounter_location": ""
+  },
+  "step1": {
+    "title": "Environmental Hygiene Safety and Counseling",
+    "fields": [
+      {
+        "key": "environment_hygiene_note_one",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "environment_hygiene_note_one",
+        "type": "label",
+        "label_text_style": "italic",
+        "text": "Hygiene and infection prevention includes appropriate hand washing, body hygiene, safe disposal of sanitary pads, washing of contaminated linens, observing hygieneic principles when handling the newborn baby and providing safe cord care."
+      },
+      {
+        "key": "environment_hygiene_spacer",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "environment_hygiene_spacer",
+        "type": "spacer"
+      },
+      {
+        "key": "environment_hygiene_note_two",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "environment_hygiene_note_two",
+        "type": "label",
+        "label_text_style": "italic",
+        "text": "I’d like to discuss disposal of children’s feces with you. A child’s stool can be dangerous and can cause diseases."
+      },
+      {
+        "key": "defecation_location",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "defecation_location",
+        "type": "edit_text",
+        "label_info_title": "Hygiene Information",
+        "hint": "Where does your child defecate?"
+      },
+      {
+        "key": "feces_disposal",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "feces_disposal",
+        "type": "edit_text",
+        "hint": "Where do you dispose your child feces?"
+      },
+      {
+        "key": "is_latrine_present",
+        "openmrs_entity_parent": "",
+        "openmrs_entity": "concept",
+        "openmrs_entity_id": "is_latrine_present",
+        "openmrs_data_type": "select one",
+        "type": "spinner",
+        "hint": "Do you have a latrine in your household",
+        "values": [
+          "Yes",
+          "No"
+        ],
+        "keys": [
+          "Yes",
+          "No"
+        ],
+        "openmrs_choice_ids": {
+          "Yes": "1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
The UCS Team is phasing out the use of openmrs uuids, please update this to the same value as key